tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -1,51 +1,135 @@
 *** Settings ***
-Documentation    Testing the backend for custom images (Adding ImageStream to redhat-ods-applications)
+Documentation    Testing custom image imports (Adding ImageStream to redhat-ods-applications)
 Resource         ../../Resources/ODS.robot
 Resource         ../../Resources/Common.robot
 Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
 Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
 Resource         ../../Resources/Page/ODH/JupyterHub/GPU.resource
 Library          JupyterLibrary
 Library          OpenShiftCLI
-Suite Teardown   Custom Image Teardown
+Library          OpenShiftLibrary
+Suite Teardown   End Web Test
 Force Tags       JupyterHub
 
 
+
 *** Variables ***
 ${YAML} =         tests/Resources/Files/custom_image.yaml
-${IMG_NAME} =     custom-test-image
+${IMG_NAME} =            custom-test-image
+${IMG_URL} =             quay.io/thoth-station/s2i-lab-elyra:v0.1.1
+${IMG_DESCRIPTION} =     Testing Only This image is only for illustration purposes, and comes with no support. Do not use.
+&{IMG_SOFTWARE} =        Software1=x.y.z
+&{IMG_PACKAGES} =        elyra=2.2.4    foo-pkg=a.b.c
+# Place holder for the imagestream name of BYON notebook created for this test run
+${IMAGESTREAM_NAME}=
 
 
 *** Test Cases ***
+Verify Admin User Can Access Custom Notebook Settings
+    Set Library Search Order  SeleniumLibrary
+    Launch Dashboard    ocp_user_name=${TEST_USER.USERNAME}    ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+    Open Notebook Images Page
+
 Verify Custom Image Can Be Added
-    [Documentation]    Applies the YAML and Gets the ImageStream
+    [Documentation]    Imports the custom image via UI
     ...                Then loads the spawner and tries using the custom img
     [Tags]    Tier2
     ...       ODS-1208
     Apply Custom ImageStream And Check Status
     Get ImageStream Metadata And Check Name
-    Begin Web Test
+    Verify Image Name  ${IMG_NAME}
```

Comment:
name suggestion: `Verify Image Is Displayed` (or present, or listed, or shown,..)